packages/eqaf/eqaf.0.6/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+name:         "eqaf"
+maintainer:   [ "Romain Calascibetta <romain.calascibetta@gmail.com>" ]
+authors:      [ "Romain Calascibetta <romain.calascibetta@gmail.com>" ]
+homepage:     "https://github.com/mirage/eqaf"
+bug-reports:  "https://github.com/mirage/eqaf/issues"
+dev-repo:     "git+https://github.com/mirage/eqaf.git"
+doc:          "https://mirage.github.io/eqaf/"
+license:      "MIT"
+synopsis:     "Constant-time equal function on string"
+description: """
+This package provides an equal function on string in constant-time to avoid timing-attack with crypto stuff.
+"""
+
+build: [
+  [ "dune" "subst" ] {pinned}
+  [ "dune" "build" "-p" name "-j" jobs ]
+  [ "dune" "runtest" "-p" name "-j" "1" "--no-buffer" "--verbose" ] {with-test}
+]
+
+depends: [
+  "ocaml"          {>= "4.03.0"}
+  "dune"
+  "alcotest"       {with-test}
+  "crowbar"        {with-test}
+]
+
+depopts: [
+  "cstruct" {>= "4.0.0"}
```

Comment:
Constraints should not be in the `depopts` field. Here it means "i'll treat this package as a dependency if `cstruct >= 4.0.0` is installed (*usually* not what you want), but if you remove the constraints and add a conflict field like this `conflicts: ["cstruct" {< "4.0.0"}]` it means "cstruct is an optional dependency and if it is installed, this packages won't work with cstruct < 4.0.0"